anchore_engine/services/policy_engine/engine/policy/gates/vulnerabilities.py

```diff
@@ -382,17 +389,19 @@ def evaluate(self, image_obj, context):
 
                                 if is_fix_available is not None:
                                     # Must do a fix_available check
-                                    fix_available_in = vulnerability_cpe.get_fixed_in()
+                                    # fix_available_in = vulnerability_cpe.get_fixed_in()  # already initialized at the top
 
                                     # explicit fix state check matches fix availability
                                     if is_fix_available == (
                                         fix_available_in is not None
                                         and len(fix_available_in) > 0
                                     ):
-                                        if is_fix_available:
-                                            parameter_data["fixed_version"] = ", ".join(
-                                                fix_available_in
-                                            )
+                                        # TODO this instantiates fix version later used for the output message only if the approrpiate gate parameter checks pass. check with Zach why that's necessary. why can't the fix version be made available regardless of the gate params
```

Comment:
IIRC this was added to ensure parameters in the rule are clear when viewing the result. However, this current impl is not good and we'll overhaul the approach when we update the policy evaluation API response. It should be safe to include this regardless.